src/CoreTests/CustomObjects/CustomPropertyTests.cs

```diff
@@ -75,5 +78,33 @@ public void ElementPropertyTest()
             prop.ConfigType = "element";
             Assert.AreEqual(CustomUIAPropertyType.Element, prop.Type);
         }
+
+        [TestMethod, Timeout(1000)]
+        public void EnumPropertyTest()
+        {
+            CustomProperty prop = new CustomProperty();
+            prop.ConfigType = "enum";
```

Comment:
Should we be using CustomProperty.*ConfigType in these tests? (Lines 38, 46, 54, 62, 70, 78, 86, 94, 106)